pypesto/visualize/sampling.py

```diff
@@ -1,18 +1,38 @@
 import logging
 import matplotlib.pyplot as plt
 import matplotlib.axes
+from matplotlib.lines import Line2D
 import numpy as np
 import pandas as pd
 import seaborn as sns
-from typing import Sequence, Tuple
-
+from typing import Dict, Sequence, Tuple, Union
+
+from .misc import (
+    rgba2rgb,
+    LEN_RGB,
+    RGBA_BLACK,
+    RGBA_MIN,
+    RGBA_MAX,
+)
 from ..result import Result
-from ..sample import McmcPtResult, calculate_ci, \
-    evaluate_samples, calculate_prediction_profiles
+from ..sample import McmcPtResult, calculate_ci
+
+from ..ensemble import (
+    get_percentile_label,
+    EnsemblePrediction,
+    MEDIAN,
+)
 
 logger = logging.getLogger(__name__)
 
 
+# TODO observable here refers to the output of an ensemble prediction, which
```

Comment:
is this still TODO or for later?

Reply:
This is because the current prediction code names all output columns `observable_ids` (e.g. accessed with `PredictionResult.conditions[0].observable_ids`), regardless of whether the post-processor only returns states. Not sure if this might be confusing to users. Any opinion @paulstapor ? Can leave as-is, and remove the TODO. I used `trajectories` in the `sampling_prediction_trajectories` function, instead of observables, to avoid confusion there. Other alternatives could be
- variables / variable_ids
- timecourses / timecourse_ids

Reply:
Also perhaps `pypesto.prediction` could be renamed to `pypesto.predict`, to match the other pyPESTO submodules

Reply:
Leaving the former point to Paul, on the latter: Yes, we use infinitive verbs for module names (and functions). Reason: Don't remember, probably shorter + sounded better.

Reply:
One could also go for "output_ids"?

Reply:
Renaming is totally fine for me
Yes, in ensemble and prediction, there are many things to be settled still. The main functionality is there, but many things may still converge to some formulation which suits everybody most